app/controllers/rules_of_use_controller.rb

```diff
@@ -0,0 +1,49 @@
+class RulesOfUseController < ApplicationController
+  before_action :confirm_signed_in
+  before_action :confirm_need_to_accept_rules_of_use
+
+  def new
+    analytics.track_event(Analytics::RULES_OF_USE_VISIT)
+    @rules_of_use_form = new_rules_of_use_form
+    render :new, locals: { request_id: nil }, formats: :html
```

Comment:
would it be easier to just have the template use `@request_id`? then we could drop this `render` line entirely and let the rails default do it's thing?